telemetry/definitions/commonDefinitions.json

```diff
@@ -92,6 +92,11 @@
             ],
             "description": "The current state of the App Runner service"
         },
+        {
+            "name": "appTemplateChoice",
```

Comment:
there is already an "action" type, and others as well. why is a new type needed?